### ISSUE TITLE
improve documentation

### ISSUE THREAD

**Issue by aspiers** [aspiers/git-deps#69]
In #39 @waldyrious made a bunch of excellent suggestions about how to improve the documentation.  This issue is for tracking the associated work.

**Comment by waldyrious** [comment]
Glad you liked the suggestions :) I think it would help, first of all, to know your thoughts about the first two points in [this comment](https://github.com/aspiers/git-deps/issues/39#issuecomment-293411821), and to [this comment](https://github.com/aspiers/git-deps/issues/39#issuecomment-293414075) as well.

**Comment by aspiers** [comment]
I agree 100% with everything you wrote, including those three points :-) I don't think I have any additional thoughts to add to them off the top of my head.

**Comment by waldyrious** [comment]
Cool, thanks. I'm a bit swamped at the moment so I don't know when I'll get the chance to tackle this, but I think there's now enough concrete actions that someone else can jump in if there's interest :)
